spacepy/pycdf/istp.py

```diff
@@ -492,6 +492,15 @@ def _validhelper(cls, v, rng=True):
                     continue
                 if firstdim: #Add pseudo-record dim
                     attrval = numpy.reshape(attrval, (1, -1))
+            if (v.attrs.type(which) in spacepy.pycdf.lib.timetypes) != (v.type() in spacepy.pycdf.lib.timetypes):
```

Comment:
Can we just add this as an "or" to the below if statement and call it good, since the error message is the same and all?